tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = black,flake8,py37
 deps =
     -rrequirements.txt
     -rrequirements-web.txt
+    gitpython
```

Comment:
This seems to be already included in requirements.txt; why are we requesting it here again?